tests/ui/regex.stderr

```diff
@@ -1,171 +1,26 @@
+   | ^^^^^^^^^^^^^^^^^^^
    |
-   = note: `-D clippy::trivial-regex` implied by `-D warnings`
-   = help: the regex is unlikely to be useful as it is
-
-error: trivial regex
-  --> $DIR/regex.rs:14:60
-   |
-LL |     let pipe_in_wrong_position_builder = RegexBuilder::new("|");
-   |                                                            ^^^
-   |
-   = help: the regex is unlikely to be useful as it is
-
-error: regex syntax error: invalid character class range, the start must be <= the end
-  --> $DIR/regex.rs:15:42
-   |
-LL |     let wrong_char_ranice = Regex::new("[z-a]");
-   |                                          ^^^
-   |
-   = note: `-D clippy::invalid-regex` implied by `-D warnings`
-
-error: regex syntax error: invalid character class range, the start must be <= the end
-  --> $DIR/regex.rs:16:37
-   |
-LL |     let some_unicode = Regex::new("[é-è]");
-   |                                     ^^^
-
-error: regex syntax error on position 0: unclosed group
-  --> $DIR/regex.rs:18:33
-   |
-LL |     let some_regex = Regex::new(OPENING_PAREN);
-   |                                 ^^^^^^^^^^^^^
-
-error: trivial regex
-  --> $DIR/regex.rs:20:53
-   |
-LL |     let binary_pipe_in_wrong_position = BRegex::new("|");
-   |                                                     ^^^
-   |
-   = help: the regex is unlikely to be useful as it is
-
-error: regex syntax error on position 0: unclosed group
-  --> $DIR/regex.rs:21:41
-   |
-LL |     let some_binary_regex = BRegex::new(OPENING_PAREN);
-   |                                         ^^^^^^^^^^^^^
-
-error: regex syntax error on position 0: unclosed group
-  --> $DIR/regex.rs:22:56
-   |
-LL |     let some_binary_regex_builder = BRegexBuilder::new(OPENING_PAREN);
-   |                                                        ^^^^^^^^^^^^^
-
-error: regex syntax error on position 0: unclosed group
-  --> $DIR/regex.rs:34:37
-   |
-LL |     let set_error = RegexSet::new(&[OPENING_PAREN, r"[a-z]+/.(com|org|net)"]);
-   |                                     ^^^^^^^^^^^^^
-
-error: regex syntax error on position 0: unclosed group
-  --> $DIR/regex.rs:35:39
-   |
-LL |     let bset_error = BRegexSet::new(&[OPENING_PAREN, r"[a-z]+/.(com|org|net)"]);
-   |                                       ^^^^^^^^^^^^^
-
-error: regex syntax error: unrecognized escape sequence
-  --> $DIR/regex.rs:37:45
-   |
-LL |     let raw_string_error = Regex::new(r"[...//...]");
-   |                                             ^^
-
-error: regex syntax error: unrecognized escape sequence
-  --> $DIR/regex.rs:38:46
-   |
-LL |     let raw_string_error = Regex::new(r#"[...//...]"#);
-   |                                              ^^
-
-error: trivial regex
-  --> $DIR/regex.rs:42:33
-   |
-LL |     let trivial_eq = Regex::new("^foobar$");
-   |                                 ^^^^^^^^^^
+note: candidate #1: /Users/YusukeAbe/code/rust/rust-clippy/target/debug/deps/libregex-a27846278b6a956a.rlib
```

Comment:
Hm, it depends on your environment. Shouldn't be changed.
Maybe: `cargo clean; cargo test` resolves this failure.

Reply:
`cargo update` sometimes fixes this for me as well

Reply:
I usually do `rm target/debug/deps/libregex*`.

Reply:
Thanks for the review. I will fix this.